### DISCUSSION TITLE
Log file location

### DISCUSSION THREAD

**Discussion by scadaguru** [tsightler/ring-mqtt#418] [Support Q&A]
I am using docker compose and I wonder how do I force logs to go into file(s) to hots mapped volume/folder, any example appreciate please, thanks

**Comment by tsightler** [comment]
Logs from ring-mqtt are just sent via stdout/stderr, so you can redirect them using any normal method.  Normal way to view them is with "docker logs" command, but if you really want them to a file you can just redirect stdout/stderr using any method common to docker, there are some good examples available online just a Google search a way, here's just one example:

https://www.sentinelone.com/blog/how-to-redirect-docker-logs-to-a-single-file/

Admittedly, that example does not use Docker compose, but the overall technique is similar.

**Comment by scadaguru** [comment]
Thanks for the reply!
Yeah, I didn't find any good example for docker compose so I updated my compose to use syslog-ng, yes that also runs in docker using compose!
I can share my syslog-ng docker compose and logger settings of ring-mqtt in case if any one interested!
Thanks for the good project and appreciate your time and efforts!
